build_tools/packager/mustache_inheritance_packager.rb

```diff
@@ -10,6 +10,26 @@ def initialize
       @base_name = "mustache_inheritance_govuk_template-#{GovukTemplate::VERSION}"
     end
 
+    def build
```

Comment:
What are these methods doing? Why are they needed for this PR?

Reply:
Oh it looks like someone started work on the mustache inheritance stuff and it never got finished and so was never made into a tarball which means you can include it in your package.json. I just copied that from the mustache one. 

Could do this as a separate PR I guess?

Reply:
I got this totally backwards (sorry been on holiday for two weeks) it created the tarball but not the uncompressed version (all the other flavours do) which makes it easier to develop